doc/api/prev_api_changes/api_changes_3.1.0.rst

```diff
@@ -795,7 +795,7 @@ Changes in parameter names
 - The *normed* parameter to `.Axes.hist2d` has been renamed to *density*.
 - The *s* parameter to `.Annotation` (and indirectly `.Axes.annotate`) has
   been renamed to *text*.
-- The *tolerence* parameter to
+- The ** parameter to
```

Comment:
Looks like a revert gone wrong. The old parameter name was *tolerence* and we need to keep it exactly spelled like it was before the change.